### PR TITLE
Reformat the code

### PR DESCRIPTION
**Pull request by XieEDeHeiShou**
I'm learning how to use `VpnService`,  then I meet this project, and I found it full of warnings ... 
So I decide to make it less warnings to let me learn faster.

( Can you please re-package those classes? They are all under the same package... )

### CODE REVIEW

File: app/src/main/java/eu/faircode/netguard/Util.java [hunk 1]
```diff
@@ -85,6 +85,7 @@
 import androidx.appcompat.app.AlertDialog;
 import androidx.core.net.ConnectivityManagerCompat;
 
+@SuppressWarnings("WeakerAccess")
```
Comment (M66B) [inline app/src/main/java/eu/faircode/netguard/Util.java]: I am no sure what this is about.

**Comment by M66B** [comment]
I see no need to rearrange code.

Why only a non null attributes for context? BTW, non null attributes were introduced long after this project started, mainly for Kotlin.

Adding documentation is fine, but it needs to be single line comments, not comment blocks. I am aware that the convention is to use comment blocks, but IMHO they take too much space and I like to see as much as possible source code on one screen.

File: app/src/main/java/eu/faircode/netguard/Util.java (hunk 1 above)
Comment (XieEDeHeiShou) [reply app/src/main/java/eu/faircode/netguard/Util.java]: This suppress the IDE (Android studio) from warning `WeakerAccess`.
For example: a method is decleared with `public` but only used by the class itself. In this situation, that method prefer to be `private`.
In this project, all classes laies under the same package, these methods should be `default`, `package-private` or `private` but `public`.

**Comment by XieEDeHeiShou** [comment]
Should I delete unused methods and/or member fileds?

**Comment by M66B** [comment]
I am sorry, but I am not going to merge this because you have now introduced bugs that can lead to crashes. NetGuard has a stable code base and I am not going to take any risk with it given that 100 thousands of people rely on NetGuard.

Note that I am nothing going to explain what is wrong because this isn't a programming course.

This is not meant unfriendly, I just want to spent my time on other things.

**Pull request closed without merging**